website_base_multi_image/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_portal_base_multi_image,access_portal_base_multi_image,base_multi_image.model_base_multi_image_image,base.group_portal,1,0,0,0
+access_public_base_multi_image,access_public_base_multi_image,base_multi_image.model_base_multi_image_image,base.group_public,1,0,0,0
```

Comment:
You are still giving access to public and portal users to every image in the database.

As this is right now, this is a :-1: blocking security issue. I think the best fix you could apply is [something similar to what I do here for `base_custom_info`](https://github.com/OCA/server-tools/pull/492/files#diff-5127b6a68d11f02283b3b595a96a1d30R100): giving access to images by checking access to their owners.

That IMP should go direclty into `base_multi_image`, BTW.